actfw_raspberrypi/__init__.py

```diff
@@ -1 +1,5 @@
 from .display import Display
+
+from actfw_raspberrypi import _version
+
+__version__ = _version.__version__
```

Comment:
```suggestion
from ._version import __version__
```

which is pythonic?

Reply:
I'm not so confident but yours seem better.